slasher/flags/flags.go

```diff
@@ -57,4 +57,11 @@ var (
 		Usage: "Sets the span cache size.",
 		Value: 1500,
 	}
+	// HighestAttCacheSize is a flag that sets the size of span cache.
+	HighestAttCacheSize = &cli.IntFlag{
+		Name:  "highest-att-cache-size",
+		Usage: "Sets the highest attestation cache size.",
+		Value: 3000,
```

Comment:
```suggestion
		Value: 300000,
```

Reply:
no need for it to be so big as it's saved in 1k buckets